### ISSUE TITLE
Exception The parameter is incorrect. (Exception from HRESULT: 0x80070057 (E_INVALIDARG)) (Microsoft Visual Studio)

### ISSUE THREAD

**Issue by pocki** [swdotcom/swdc-visualstudio#27]
Since the update of today I am always getting this exception in an SQL Integration Service Project in Visual Studio, for example on saving a change at parameters or starting to execute a SSIS Package

I didn't have this exception with the last versions before (sorry, I don't know the version number), Project is also nearly the same for the last months.
```
The parameter is incorrect. (Exception from HRESULT: 0x80070057 (E_INVALIDARG)) (Microsoft Visual Studio)

------------------------------
Program Location:

   at EnvDTE.Document.get_TabSize()
   at SoftwareCo.DocEventManager.UpdateFileInfoMetrics(PluginDataFileInfo fileInfo, TextPoint start, TextPoint end, String Keypress, TextSelection textSelection) in C:\Users\Software\projects\swdc-visualstudio\SoftwareCo\SoftwareCo\Managers\DocEventManager.cs:line 211
   at SoftwareCo.DocEventManager.<LineChangedAsync>d__11.MoveNext() in C:\Users\Software\projects\swdc-visualstudio\SoftwareCo\SoftwareCo\Managers\DocEventManager.cs:line 113
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

Visual Studio Enterprise 2019 Version 16.7.2
Code Time Version 2.2.2

**Comment by brettmstevens7** [comment]
@pocki we're on it

**Comment by xavluiz** [comment]
Thank you again for the issue update. We've published v2.2.3 to fix that issue, along with getting the icons back into the tree view.
Regards.

**Comment by pocki** [comment]
thank you, seems to work no again. No Exceptions any more
